zendesk_attachment_backup.py

```diff
@@ -69,37 +73,51 @@ def collect_urls(input_files, print_urls, **kwargs):
 
    # Print URLs to file if requested
    if print_urls:
-      with open('attachment_urls.txt', "w") as fh:
+      target_file=os.path.join(kwargs.get("output_dir"), 'attachment_urls.txt')
```

Comment:
heh, guilty as charged. I like being able to do crazy things like this, though : https://github.com/SciLifeLab/genologics_sql/blob/master/genologics_sql/tables.py#L47

But yeah, you're right, standards are here for a reason.